share/src/main/java/org/apache/kafka/server/share/persister/PersisterStateManager.java

```diff
@@ -212,6 +212,7 @@ public abstract class PersisterStateManagerHandler implements RequestCompletionH
         private final BackoffManager findCoordBackoff;
         protected final Logger log = LoggerFactory.getLogger(getClass());
         private Consumer<ClientResponse> onCompleteCallback;
+        private SharePartitionKey partitionKey;
```

Comment:
Now when SharePartitionKey is at instance variable then we should remove grouId, topicId, partition as instance variables.